bibtex-actions.el

```diff
@@ -368,11 +368,11 @@ key associated with each one."
                (when (or (bibtex-actions-get-value
                           bibtex-actions-file-variable entry)
                          (bibtex-actions-file--files-for-key
```

Comment:
BTW
```elisp
(bibtex-actions-get-value  bibtex-actions-file-variable entry)
```
here and as a result the `or` can be removed, sicne `bibtex-actions-file--files-for-key` takes care of it now.

Reply:
Folded this into latest commit.